src/Controller/ExportController.php

```diff
@@ -113,6 +114,11 @@ public function export(Request $request): Response
 
         $renderer = $this->export->getRendererById($type);
 
+        // display file inline if supported and `markAsExported` is not set
+        if ($renderer instanceof DispositionInlineInterface && !$query->isMarkAsExported()) {
+            $renderer->setDispositionInline(true);
+        }
+
         if (null === $renderer) {
```

Comment:
Probably the new code should go below the check for null

Reply:
You're totally right. I did not realize this check. 😅